src/Symfony/Component/Process/Process.php

```diff
@@ -65,6 +65,14 @@ class Process
     private $latestSignal;
 
     private static $sigchild;
+    private static $posixSignals = array(
```

Comment:
for the reader: this is the list of signals that are standardize per posix (except 9, SIGKILL, which can't be caught, thus forwarded)